### ISSUE TITLE
Add info about docker in Readme

### ISSUE THREAD

**Issue by AdiNar** [Stock-Engine/stock-engine-frontend#32]
Currently frontend can be deployed using docker, expecting api on 8008.

**Comment by Abhay2412** [comment]
Hi, I would like to give a shot at working on this and some more information

**Comment by AdiNar** [comment]
Hi @Abhay2412, thanks for your involvement! 

I'll write some more detailed system description after I defend my master thesis this week. In short, Every merged PR triggers a build of `adinar/stock-engine-frontend:latest` docker image, and we'll manually choose builds for the `stable` tag. Those can be deployed using `docker-compose-staging.yml` and `docker-compose.yml`. BTW, those are also automatically deployed to [https://staging-se.naruszko.dev/](https://staging-se.naruszko.dev/) and [https://stockengine.naruszko.dev/](https://stockengine.naruszko.dev/) (although not much is there to see now ;) ). Only the frontend is dockerized currently, backend mock (see api/ directory) is yet to be dockerized. 

To sum up: we would like to inform users that 1) there are docker images and that 2) they can be deployed using `docker-compose*` files and that 3) backend is not dockerized yet, instead it is expected to run on port 8008.
